### PR TITLE
 [Docs] Chinese/Mandarin version of Aptos Whitepaper 

### PR DESCRIPTION
**Pull request by ruy1su**
### Description
Add chinese/mandarin version of Aptos whitepaper to aptos dev webpage.

### Test Plan
Mostly adding docs and new paper pdf version. Please let me know if I should add any test.

- - -
This change is [Reviewable](https://reviewable.io/reviews/aptos-labs/aptos-core/3762)

### CODE REVIEW

**Comment by davidiw** [comment]
@msmouse , @zekun000 , @vgao1996 can you take a look at this and provide feedback?

File: developer-docs-site/docs/aptos-white-paper/in-chinese.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+区块链是一种新的互联网基础设施，随着区块链的兴起，开发人员以快速增长的速度部署了数以万计的去中心化应用程序。
+不幸的是，由于频繁中断、高成本、低吞吐量限制和众多安全问题，区块链的使用尚未普及。为了在web3时代实现大规模应用，
+区块链基础设施需要借鉴云基础设施的路径，成为构建广泛使用的应用程序的可信、可扩展、具有成本效益且不断改进的平台。
+
+我们展示了以可扩展性、安全性、可靠性和可升级性为关键原则设计的Aptos区块链，用以应对这些挑战。Aptos区块链在过去
+三年中由全球超过350名开发人员设计开发。它在共识、智能合约设计、系统安全、性能和去中心化方面提供了新的创新。
+这些技术的结合将为web3推向大众提供一个基本的构建块：
+
+- 第一，Aptos区块链在本地集成并在内部使用Move语言，以实现快速安全的交易执行。Move prover是用Move语言编写的
+智能合约形式化验证器，它为合约不变性和行为提供了额外的保障。这种对安全性的关注使得开发人员能够更好地保护他们的
```
[the remark this answers is not in the document]
Comment (vgao1996) [reply developer-docs-site/docs/aptos-white-paper/in-chinese.md:20]: This change is essential as the original text says the Prover is written in Move, which is not true.

**Comment by davidiw** [comment]
What about the PDF itself?

**Comment by vgao1996** [comment]
@davidiw my bad, thought this was only a translation of the web page ;)

Will have another look at the paper

**Comment by ruy1su** [comment]
Thanks for taking a look, team!

File: developer-docs-site/docs/aptos-white-paper/in-chinese.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (ruy1su) [reply developer-docs-site/docs/aptos-white-paper/in-chinese.md]: Absolutely. Updated and thanks for the correction :D

**Comment by davidiw** [comment]
> Thanks for taking a look, team!

Thank you for contributing!

**Comment by ChengYueJia** [comment]
@ruy1su Thanks for ur hard-working.

There are many zh_cn version Aptos-version.
For me, many of there are too hurry to be translated.

To avoid this, may be the team @davidiw  take the markdown part.
I believe there will be more and more guys to make it better.

Hope this can help all u guys.

**Comment by vgao1996** [comment]
I took a look at the Move parts (chap 4 & 5) and made some comments (see the annotated pdf below). 
[whitepaper-chinese-annotated.pdf](https://github.com/aptos-labs/aptos-core/files/9499731/whitepaper-chinese-annotated.pdf)

**Comment by vgao1996** [comment]
I took a look at the Move parts (chap 4 & 5) and made some comments (see the annotated pdf below). 
[whitepaper-chinese-annotated.pdf](https://github.com/aptos-labs/aptos-core/files/9499747/whitepaper-chinese-annotated.pdf)

**Comment by ruy1su** [comment]
> I took a look at the Move parts (chap 4 & 5) and made some comments (see the annotated pdf below). [whitepaper-chinese-annotated.pdf](https://github.com/aptos-labs/aptos-core/files/9499747/whitepaper-chinese-annotated.pdf)

Thanks for the detailed annotation. That helps a lot! I will try to make another revision today :D

**Comment by ruy1su** [comment]
> @ruy1su Thanks for ur hard-working.
> 
> There are many zh_cn version Aptos-version I've seen. Many of them are translated a little hurry. There are a lot space to be better.
> 
> To avoid this, may be the team @davidiw take the markdown part. I believe there will be more and more guys to make it better.
> 
> Hope this can help all u guys.

Thanks! So do you think it's better to remove markdown files for now?

**Comment by ChengYueJia** [comment]
@ruy1su Sorry, maybe you mistake my view.

 It's better to keep the markdown. It's easier to manage for the white paper and for the dev docs.
* First, We can modify the translation.
*  Second, later the dev docs can be translated by markdown.

Besides, do we have plan to i18n the dev docs? @davidiw

**Comment by ruy1su** [comment]
> @ruy1su Sorry, maybe you mistake my view.
> 
> It's better to keep the markdown. It's easier to manage for the white paper and for the dev docs.
> 
> * First, We can modify the translation.
> * Second, later the dev docs can be translated by markdown.
> 
> Besides, do we have plan to i18n the dev docs? @davidiw

Absolutey! If we keep using the markdown format, we probably have to change the latex version we have now and that would take a while - let me know if that's a blocker to this PR. On the other hand, I think that's a two-way door. That means -- We can alway add markdown format of whitepaper later and keep the pdf/latex version of white paper for folks who would like download, reference or share to other entity/users. Using PDF is not a bad idea and Korean community contributed their version which I think it's a great start. @vgao1996 Hey Victor, let me know if there's any other blocker for this PR and thank you for annotated comments! Appreciate y'all for the feedback :D!

**Comment by davidiw** [comment]
Looks like we're moving the content to here: https://github.com/aptos-labs/whitepaper/tree/main/mandarin

Happy to receive contributions over there

**Pull request closed without merging**